### PR TITLE
Move to express

### PR DESCRIPTION
**Pull request by jaumard**
@tjwebb here is the migration to express that can be use to 4 and 5. Can you rename the repo to `trailpack-express` ? Remove or disable `trailpack-express4` from npm and deploy this one (but let me know to update `generator-trailpack` at the same time ? 

Do you have an idea of how I can unit test for both express 4 and 5 ?

### CODE REVIEW

**Comment by tjwebb** [comment]
@jaumard we can set up a build matrix, and define the express version in an environment variable. e.g. `EXPRESS_VERSION=4` and then manually `npm install express@${EXPRESS_VERSION}`

Is the generator-trailpack PR ready? If so, send me the link to it, and I will rename, merge, and publish a new version at the same time.

**Comment by jaumard** [comment]
@tjwebb Mocha can do build matrix ? Because don't want to test only on Travis but also on local. But I will look this way thanks! generator-trails is not ready yet but it's not too long, I'm doing this today. Keep you updated :)

**Comment by jaumard** [comment]
@tjwebb the PR for generator-trails is here https://github.com/trailsjs/generator-trails/pull/44, I added on this PR test agains Express4, Express@latest and express@5alpha2. Travis fails because of node@6 :/

**Comment by jaumard** [comment]
It will be better to do it this way : 
- Merge this one and publish a npm version under `trailpack-express`
- Rename the repo to `trailpack-express`
- Make sure the PR on `generator-trails` work great with the new `trailpack-express`
- Merge the PR and publish `generator-trails`
- Remove/disable `trailpack-express4` from npm

What do you think @tjwebb  ?

File: .travis.yml [hunk 1]
```diff
@@ -18,5 +19,5 @@ deploy:
     secure: J8ZYI+2vEbILgSalnTi1wZj374kTa+6yY67SW7uMBDKx6AQprN1Qxc00B7OmrI90xTo6Ia2wSzFPjsiPbh8BHnLI8Eoz0SoHAa5iPORx5XztLJN6kQsjeQWNuCBP5wjvDtsYlJLJsNppjun0Ywnl0w6rsFIBfcbHYRHrovntB8qiuGY5ZSq9Rnki0oDgilwIMLCUlOx1ajB+c6oRq59cNn5TKbuGh7Etb1plzsuPLDkrjrtaw9RcDp49P6jJFNCSsKNJgpy8Ki7QRVA52l16SfrRfctAO98NR6+Y+0kjUqbm+CEcAtMnzmF7/Nqi1VLfP9tRYwR0HbNh4X9yrzfOu1ilF8ZHedB0LPaAu0GdTbPJwkXVweueDQJlAYY5X8rcPbXEYUo5aEL082UhSreVFO/Q7FaG+JmbDqnMDzj2cS2zV1ruPfv8C319rSUgKRFe4GZMUcttbqZfAz490W7Y3PAyBeS7gjG8fxGyv/5iq42JYC60CZIllVLxinwRzAff3zSf8xUhCIZFjGiNb9qGaEj8jo9/vNkUfnCGFrshMEmRqUjS6ae/wS5qPlJ4pCfNCMQDNAwbDeVk5Tc9FmPRtd1lA7j63vAqe9zsKnaMzBmYAPY+wEMvOl8n+SxAylKLy+v44H8MX7nykd58qwdOW8kyYWJqAK7QrwxgnvD8n5Y=
   on:
     tags: true
-    repo: trailsjs/trailpack-express4
+    repo: trailsjs/trailpack-express
```
Comment (tjwebb) [inline .travis.yml]: @jaumard FYI, due to the way Travis computes the hashes for their secure keys (line 19), changing this line will usually cause the key to fail. I'll submit this change in a separate PR.

Comment (jaumard) [inline .travis.yml]: Ho you're right ! Thanks !

**Comment by tjwebb** [comment]
@jaumard I'm going to to ahead and merge this, and then I'll make the TravisCI changes.

Also, I renamed the repo.

**Pull request merged**